### ISSUE TITLE
better handle errors during vivisect load

### ISSUE THREAD

**Issue by williballenthin** [fireeye/flare-floss#179]
users don't want to see tracebacks, they want to be told whats wrong, and what to do about it.

for example, take the following: 

```
$ file 82afe1bd-878e-5b27-b4f9-636393a3fbcb
82afe1bd-878e-5b27-b4f9-636393a3fbcb: MS-DOS executable

$ floss -v 82afe1bd-878e-5b27-b4f9-636393a3fbcb
INFO:floss:Generating vivisect workspace
Traceback (most recent call last):
  File "/usr/local/bin/floss", line 9, in <module>
    load_entry_point('floss==1.2.0', 'console_scripts', 'floss')()
  File "/usr/local/lib/python2.7/dist-packages/floss-1.2.0-py2.7.egg/floss/main.py", line 546, in main
    vw = viv_utils.getWorkspace(sample_file_path)
  File "/usr/local/lib/python2.7/dist-packages/viv_utils-0.3.2-py2.7.egg/viv_utils/__init__.py", line 35, in getWorkspace
    vw.loadFromFile(fp)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/__init__.py", line 2134, in loadFromFile
    fname = mod.parseFile(self, filename)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/parsers/pe.py", line 30, in parseFile
    return loadPeIntoWorkspace(vw, pe, filename)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/parsers/pe.py", line 65, in loadPeIntoWorkspace
    raise Exception("Machine %.4x is not supported for PE!" % mach )
Exception: Machine 0401 is not supported for PE!
```

it would be much nicer to see:

```
$ floss -v 82afe1bd-878e-5b27-b4f9-636393a3fbcb
INFO:floss:Generating vivisect workspace
ERROR:floss:FLOSS does not support this type of PE file (Machine: 0401, MS-DOS)
```

since vivisect doesn't define a lot of custom exception classes, we'll have to handle these errors on a one-off case as reports come in. maybe do something like `if "not support for PE" in str(e): ...`

**Comment by williballenthin** [comment]
cc: @pilate

**Comment by williballenthin** [comment]
closed in #180
